Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +8,18 @@ class Program
     {
         static void Main(string[] args)
         {
+            for (int i = 0; i < 2; i++)
+            {
+                var surname = HwFunctions.PromptString("Please enter your surname: ");
+                var age = HwFunctions.PromptInt("Please enter your age: ");
+                var weight = HwFunctions.PromptFloat("Please enter your weight (in kg): ");
+                var height = HwFunctions.PromptFloat("Please enter your height (in m): ");
+                Console.WriteLine(surname + " is " + age + " " + "years old, his weight is " + weight + "kg and his height is " + height + "m.");
+                
+                var Bmi = HwFunctions.CalculateBmi(weight, height);
+                Console.WriteLine("Your estimated BMI is " + Bmi);
+            }
```

Comment:
That should be the code in the method named "Demo()" of "Lesson3" class.

Reply:
Updated homework to reflect Discord instructions, not the homework instructions from Twitch chat. Thanks!